src/firestore-adapter.ts

```diff
@@ -678,20 +671,17 @@ export class FirestoreAdapter implements IDatabaseAdapter {
    * Attach listeners for `child_added`, `child_changed` and `child_removed` event on `users` node of Firebase ref.
    */
   protected _monitorCursors(): void {
-    const firestoreUsersRef = this._firestoreRef!.collection("users");
+    const firestoreUsersRef = this._firestoreRef!.collection("users").where(
+      "lastUpdated",
+      ">=",
+      Date.now() - FirestoreAdapter.MAX_CURSOR_SYNC_TIME
```

Comment:
are there any updates we might miss with this? 
For example, previous behavior used to send all "removed" updates

Reply:
The original firebase implementation before lastUpdated would not send any removed cursors on first retrieval since they wouldn't be there in the database in first place.

And the new removals will still be monitored with this query.

Not an issue 👍🏼